service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -170,7 +170,7 @@ private void sendC21NotificationForLocalAuthority(final CaseDetails caseDetails,
         String localAuthorityEmail = localAuthorityEmailLookupConfiguration
             .getLocalAuthority(localAuthorityCode)
             .map(LocalAuthorityEmailLookupConfiguration.LocalAuthority::getEmail)
-            .orElse("");
+            .orElseThrow(() -> new NullPointerException("Expected to have local authority email"));
```

Comment:
It doesn't mimic the old message, I don't mind but we should see what's the local authority code to make investigations easier. Please add at least the code as in previous message:
"Local authority '" + localAuthorityCode + "' not found"

Reply:
Done this change